node/cli/src/chain_spec.rs

```diff
@@ -185,25 +244,23 @@ pub fn darwinia_genesis(
 ) -> GenesisConfig {
 	let endowed_accounts: Vec<AccountId> = endowed_accounts.unwrap_or_else(|| {
 		vec![
-			get_account_id_from_seed::<sr25519::Public>("Alice"),
-			get_account_id_from_seed::<sr25519::Public>("Bob"),
+			initial_authorities[0].clone().1,
+			initial_authorities[1].clone().1,
 			get_account_id_from_seed::<sr25519::Public>("Charlie"),
 			get_account_id_from_seed::<sr25519::Public>("Dave"),
 			get_account_id_from_seed::<sr25519::Public>("Eve"),
 			get_account_id_from_seed::<sr25519::Public>("Ferdie"),
-			get_account_id_from_seed::<sr25519::Public>("Alice//stash"),
-			get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
+			initial_authorities[0].clone().0,
+			initial_authorities[1].clone().0,
 			get_account_id_from_seed::<sr25519::Public>("Charlie//stash"),
 			get_account_id_from_seed::<sr25519::Public>("Dave//stash"),
 			get_account_id_from_seed::<sr25519::Public>("Eve//stash"),
 			get_account_id_from_seed::<sr25519::Public>("Ferdie//stash"),
 		]
 	});
 
-	let eth_relay_authorities: Vec<AccountId> = vec![
-		get_account_id_from_seed::<sr25519::Public>("Alice"),
-		get_account_id_from_seed::<sr25519::Public>("Bob"),
-	];
+	let eth_relay_authorities: Vec<AccountId> =
+		vec![initial_authorities[0].clone().1, initial_authorities[1].clone().1];
```

Comment:
Also here.

Reply:
A flag `is_test_net` is used to check the config is used under a test net or not.
In test net, the config will use original random generate keys for Alice and Bob.